libs/clients/althingi-ombudsman/src/lib/const.ts

```diff
@@ -0,0 +1,2 @@
+export const SECURITY_TOKEN_KEY = 'securityToken'
```

Comment:
~My concern here would be that this key is to generic and should be something like `althingiOmbudsmanSecurityToken`